browser/about_flags.cc

```diff
@@ -316,6 +320,13 @@ constexpr char kFileSystemAccessAPIDescription[] =
      kOsDesktop | flags_ui::kOsAndroid,                                      \
      FEATURE_VALUE_TYPE(brave_wallet::features::kNativeBraveWalletFeature)},
 
+#define BRAVE_NEWS_FEATURE_ENTRIES                                  \
+    {"brave-news",                                                  \
+     flag_descriptions::kBraveNewsName,                             \
+     flag_descriptions::kBraveNewsDescription,                      \
+     kOsDesktop | flags_ui::kOsAndroid,                             \
```

Comment:
nit: other feature entries do not specify `flags_ui`

Reply:
it's there for the wallet https://github.com/brave/brave-core/blob/1c782afe06132508c64eec77130e71465ba530e2/browser/about_flags.cc#L315